Hw_03/hw03.Rmd

```diff
@@ -0,0 +1,205 @@
+---
+title: "HW03"
+author: "Tyler Allan"
+date: "2/18/2021"
+output: 
+    github_document: default 
+    html_document: default 
+    keep_md: TRUE
+---
+# Homework 03
+
+
+
+You will analyze data looking at the relationship between **green reputation** and three personality traits--**compassion**, **intellectual curiosity**, and **openness to experiences**.
+The dataset includes data from **students** and **non-students**.
+
+```{r}
+dictionary <- readr::read_csv("green_dictionary.csv")
+green_data <- readr::read_csv("green_data.csv")
+library(tidyverse)
+```
+
+For your assignment, do the following.
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+
+```{r}
+print(green_data)
+summary(green_data$comp1)
+
+new_green_data <- green_data %>% 
+                na_if("-99")
+print(new_green_data)
+summary(new_green_data$green1)
+```
+
+  2. Compute total scores for the four scales. 
+     Recode variables as needed.
+     
+```{r}
+reversed <- c("green2", "comp6", "comp7", "comp8", "comp9", "comp10", 
+              "intel7", "intel8", "intel9", "intel10", "open7", "open8", 
+              "open9", "open10")
+new_green_datar <- new_green_data %>% 
+                    mutate(
+                           across(all_of(reversed),
+                           ~ recode(.x, "1" = 5, "2" = 4, "4" = 2, "5" = 1),
+                           .names = "{.col}r")
+                          ) 
+
+new_green_datar <- new_green_datar %>% 
+  rowwise() %>% 
+  mutate(green_tot = sum(c(green1,green2r,green3,green4,green5), na.rm = TRUE),
+         comp_tot = sum(c(comp1,comp2,comp3,comp4,comp5,comp6r,comp7r,comp8r,comp9r,comp10r), na.rm = TRUE),
+         intel_tot = sum(c(intel1,intel2,intel3,intel4,intel5,intel6,intel7r,intel8r,intel9r,intel10r), na.rm = TRUE),
+         open_tot = sum(c(open1,open2,open3,open4,open5,open6,open7r,open8r,open9r,open10r), na.rm = TRUE),) 
```

Comment:
Are we supposed to create a sum or a mean score? You could be right; I'm not sure.

Reply:
I was also unsure. I started by computing the mean, but the question asks for a total score not an average score and that's why I went with sum.